src/libraries/System.DirectoryServices.AccountManagement/tests/System.DirectoryServices.AccountManagement.Tests.csproj

```diff
@@ -12,10 +12,12 @@
     <Compile Include="AccountManagementTests.cs" />
     <Compile Include="$(CommonTestPath)System\DirectoryServices\LdapConfiguration.cs"
              Link="Common\DirectoryServices\LdapConfiguration.cs" />
+	  <Compile Include="$(CommonTestPath)TestUtilities\System\SecretsHelper.cs"
+             Link="Common\TestUtilities\System\SecretsHelper.cs" />
   </ItemGroup>
   <ItemGroup>
     <Content Include="LDAP.Configuration.xml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Nit: end line white space diff.

Reply:
I added new line to be consistent with other csproj files